daprdocs/content/en/reference/components-reference/supported-state-stores/_index.md

```diff
@@ -9,10 +9,12 @@ aliases:
 no_list: true
 ---
 
-{{< partial "components/description.html" >}}
+The following table lists state stores supported, at various levels, by the Dapr state management building block. [Learn how to set up different state stores for Dapr state management.]({{< ref setup-state-store.md >}})
```

Comment:
Same comment as above. What did you mean by "at various levels"? Do you mean certification status?